docs/java-rest/high-level/search/point-in-time.asciidoc

```diff
@@ -0,0 +1,78 @@
+[[java-rest-high-search-point-in-time]]
+
+=== Open a point in time
+
+A point in time must be opened before being used in search requests.
+An OpenPointInTimeRequest requires an `index` and `keepAlive` arguments:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/SearchDocumentationIT.java[open-point-in-time]
+--------------------------------------------------
+<1> Create an `OpenPointInTimeRequest` with the target indices
+<2> Set the `keep_alive` parameter, which tells Elasticsearch how long it
```

Comment:
may be worth adding that this `keep_alive` is required parameter.

Reply:
++, adjusted in https://github.com/elastic/elasticsearch/pull/72167/commits/754a2ed9e3050894a9c9d5a4d6a8aadfc4eaa6a9